Description: Set NoDisplay=true for vim.desktop
Author: Tianyu Chen <sweetyfish@deepin.org>
Origin: vendor
Bug-deepin: https://pms.uniontech.com/bug-view-285241.html
Forwarded: not-needed
Last-Update: 2024-11-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/po/vim.desktop.in
+++ b/src/po/vim.desktop.in
@@ -70,6 +70,7 @@
 TryExec=vim
 Exec=vim %F
 Terminal=true
+NoDisplay=true
 Type=Application
 # Translators: Search terms to find this application. Do NOT change the semicolons! The list MUST also end with a semicolon!
 Keywords=Text;editor;
--- a/runtime/vim.desktop
+++ b/runtime/vim.desktop
@@ -111,6 +111,7 @@
 TryExec=vim
 Exec=vim %F
 Terminal=true
+NoDisplay=true
 Type=Application
 # Translators: Search terms to find this application. Do NOT change the semicolons! The list MUST also end with a semicolon!
 Keywords[ca]=Text;editor;
